src/types/index.ts

```diff
@@ -153,12 +155,11 @@ export interface IAccount {
 /**
  * Account fetched from the node with the use of `sdk.api.getAccountByPubkey`
  */
-export interface IAccountFetched {
+type accountFetched = Awaited<ReturnType<InstanceType<typeof Node>['getAccountByPubkey']>>;
```

Comment:
Types should use PascalCase naming. Also for the future adoption of other blockchains I suggest to add "Aeternity" to the name.